components/entities/api/src/main/java/org/phenotips/entities/internal/AbstractPrimaryEntityManager.java

```diff
@@ -268,20 +268,6 @@ protected DocumentReference getNextDocument()
         return newDoc;
     }
 
-    /**
-     * Gets a prefix for all {@code #create() generated} documents. This implementation computes it from the uppercase
-     * letters of the XClass name, excluding {@code Class}, e.g. for {@code PhenoTips.DiseaseStudyClass} the prefix will
-     * be {@code DS}.
-     *
-     * @return a short string
-     */
-    protected String getIdPrefix()
```

Comment:
I wouldn't remove this, it's an acceptable default implementation, and it can be overridden by the actual classes. Just change it to `public`.

Reply:
Fixed.